core/src/commands/get/get-actions.ts

```diff
@@ -0,0 +1,292 @@
+/*
+ * Copyright (C) 2018-2023 Garden Technologies, Inc. <info@garden.io>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import Bluebird from "bluebird"
+import chalk from "chalk"
+import { relative, sep } from "path"
+import { actionReferenceToString } from "../../actions/base"
+import { ResolvedBuildAction } from "../../actions/build"
+import { ResolvedDeployAction } from "../../actions/deploy"
+import { ResolvedRunAction } from "../../actions/run"
+import { ResolvedTestAction } from "../../actions/test"
+import { Action, ActionKind, ActionState, ResolvedAction, actionKinds, actionStateTypes } from "../../actions/types"
+import { BooleanParameter, ChoicesParameter, StringsParameter } from "../../cli/params"
+import { ResolvedConfigGraph } from "../../graph/config-graph"
+import { Log, createActionLog } from "../../logger/log-entry"
+import { printHeader } from "../../logger/util"
+import { ActionRouter } from "../../router/router"
+import { sanitizeValue } from "../../util/logging"
+import { deepFilter } from "../../util/objects"
+import { dedent, deline, renderTable } from "../../util/string"
+import { Command, CommandParams, CommandResult } from "../base"
+import { createSchema, joi, joiArray } from "../../config/common"
+import { actionConfigSchema } from "../../actions/helpers"
+
+interface GetActionsCommandResultActionObject {
```

Comment:
```suggestion
interface GetActionsCommandResultItem {
```
I think it reads better like this